README.md

```diff
@@ -156,6 +148,47 @@ Configure the form-submit functionality which enables you to submit a (login) fo
 | input_filter      | A list of input fields that should not be submitted with the form                                         | False    |         | string - list       |
 | submit_once       | Submit the form only once on startup instead of each scan interval                                        | False    | False   | boolean             |
 | resubmit_on_error | Resubmit the form after a scraping error is encountered                                                   | False    | True    | boolean             |
+| variables         | See [Form Variables](#Form-Variables)                                                                     | False    |         | list                |
+
+### Form Variables
+
+Configure the variables that will be scraped from the [`form_submit`](#form-submit) response. These variables can be used in the `value_template` of a header, a sensor or an attribute of the main configuration of the current integration. A common use case is to populate the `X-Login-Token` header which is the result of the login.
+
+| name | description                      | required | default | type   |
+| ---- | -------------------------------- | -------- | ------- | ------ |
+| name | Name of the variable             | True     |         | string |
+|      | See [Selector](#Selector) fields | True     |         |        |
+
+Example:
+
+```yaml
+multiscrape:
+  - resource: "https://somesiteyouwanttoscrape.com"
+    form_submit:
+      submit_once: True
+      resource: "https://authforsomesiteyouwanttoscrape.com"
+      input:
+        email: "<email>"
+        password: "<password>"
+      variables:
+        - name: token
+          value_template: "{{ ... }}"
+    headers:
+      X-Login-Token: "{{ token }}"
```

Comment:
@danieldotnl, when using variables, should we use them directly or should we add a prefix? For example: `form_variables.token`. This is basically wrapping all form variables into one json object/dict which should be more future proof. On the other hand, it might be overengineering. :D

Reply:
It's a very good thought, but indeed also over-engineering and adding a bit of extra complexity for the user. I think when we have more types of variables in the future, we should add a check for duplicate variable names instead. So leave it like it is for now.